### PR TITLE
dev-libs/libconfig-9999: Add ebuild to overlay

### PR DESCRIPTION
**Pull request by marbre**
The current version of libconfig in the git repository contain some minor fixes. E.g. the gentoo specific patches are not needed any longer.

### CODE REVIEW

File: dev-libs/libconfig/libconfig-9999.ebuild [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+inherit eutils autotools-multilib
+
+DESCRIPTION="Libconfig is a simple library for manipulating structured configuration files"
+HOMEPAGE="http://www.hyperrealm.com/libconfig/libconfig.html"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+if [[ ${PV} == "9999" ]] ; then
+	EGIT_REPO_URI="https://github.com/hyperrealm/libconfig.git"
+	EGIT_BRANCH="master"
```
Comment (junghans) [inline dev-libs/libconfig/libconfig-9999.ebuild]: Remove, is the default.

File: dev-libs/libconfig/libconfig-9999.ebuild [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+AUTOTOOLS_AUTORECONF="1"
+inherit eutils autotools-multilib
+
+DESCRIPTION="Libconfig is a simple library for manipulating structured configuration files"
+HOMEPAGE="http://www.hyperrealm.com/libconfig/libconfig.html"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+if [[ ${PV} == "9999" ]] ; then
+	EGIT_REPO_URI="https://github.com/hyperrealm/libconfig.git"
```
Comment (junghans) [inline dev-libs/libconfig/libconfig-9999.ebuild]: Add `git://` url, too.

File: dev-libs/libconfig/libconfig-9999.ebuild [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+	SRC_URI="http://www.hyperrealm.com/${PN}/${P}.tar.gz"
+	KEYWORDS="amd64 ~arm ~mips ppc ppc64 sparc x86 ~x86-linux"
+	PATCHES=( "${FILESDIR}/${P}-out-of-source-build.patch" )
+fi
+
+IUSE="+cxx examples static-libs"
+
+DEPEND="
+	sys-devel/libtool
+	sys-devel/bison"
```
Comment (junghans) [inline dev-libs/libconfig/libconfig-9999.ebuild]: I guess `virtual/yacc`.

Comment (marbre) [inline dev-libs/libconfig/libconfig-9999.ebuild]: Possible.. I only modified the the ebuild from the gentoo main tree a little to build with the github source. Anyway, done :)

**Comment by marbre** [comment]
Thanks for the hints. All done.

File: dev-libs/libconfig/libconfig-9999.ebuild [hunk 4]
```diff
@@ -0,0 +1,61 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+AUTOTOOLS_AUTORECONF="1"
+inherit eutils autotools-multilib
+
+DESCRIPTION="Libconfig is a simple library for manipulating structured configuration files"
```
Comment (jlec) [inline dev-libs/libconfig/libconfig-9999.ebuild]: You can drop PN from DESCRIPTION

Comment (marbre) [inline dev-libs/libconfig/libconfig-9999.ebuild]: This is copied from the portage main tree... Anyway, should I fix, commit and do a new pull request?

**Pull request merged**